birdhouse/components/stac/service-config.json.template

```diff
@@ -1,54 +1,77 @@
-{
-  "$schema": "https://raw.githubusercontent.com/DACCS-Climate/Marble-node-registry/main/node_registry.schema.json#service",
-  "name": "stac",
-  "keywords": [
-    "catalog"
-  ],
-  "description": "Searchable spatiotemporal metadata describing climate and Earth observation datasets.",
-  "links": [
-    {
-      "rel": "service",
-      "type": "application/json",
-      "href": "https://${PAVICS_FQDN_PUBLIC}/stac/"
-    },
-    {
-      "rel": "service-doc",
-      "type": "text/html",
-      "href": "https://github.com/radiantearth/stac-api-spec/tree/v1.0.0"
-    },
-    {
-      "rel": "service-desc",
-      "type": "application/yaml",
-      "href": "https://raw.githubusercontent.com/radiantearth/stac-api-spec/main/core/openapi.yaml"
-    },
-    {
-      "rel": "alternate",
-      "type": "text/html",
-      "href": "https://${PAVICS_FQDN_PUBLIC}/stac-browser/"
-    }
-  ]
-}, {
-  "$schema": "https://raw.githubusercontent.com/DACCS-Climate/Marble-node-registry/main/node_registry.schema.json#service",
-  "name": "stac-browser",
-  "keywords": [
-    "catalog"
-  ],
-  "description": "A web interface to browse the STAC catalogs.",
-  "links": [
-    {
-      "rel": "service",
-      "type": "text/html",
-      "href": "https://${PAVICS_FQDN_PUBLIC}/stac-browser/"
-    },
-    {
-      "rel": "service-doc",
-      "type": "text/html",
-      "href": "https://github.com/radiantearth/stac-browser"
-    },
-    {
-      "rel": "alternate",
-      "type": "application/json",
-      "href": "https://${PAVICS_FQDN_PUBLIC}/stac/"
-    }
-  ]
-}
+[
```

Comment:
I don't understand why this change is necessary, it makes the format inconsistent with the rest of these files and makes parsing them more complicated (see other comment).

If the answer is that you want this file to be valid json ... I don't think that is a good enough reason to change this, this file is for internal use only and isn't meant to be read by any external client without being processed first.

Reply:
The format is also inconsistent currently. It is not a vaild JSON, so it cannot even be parsed unless hacking it before load.

Reply:
I prefer the all list approach since it works for every use case.